src/components/navbar/Navbar.js

```diff
@@ -1,102 +1,101 @@
-import React, { Component, Fragment } from 'react'
-import { Menu, Container, Image, Icon } from 'semantic-ui-react'
-import { Link } from 'react-router-dom'
-import { withRouter } from 'react-router'
-import logo from '../../images/av-logo.svg'
-import './Navbar.css'
+import React, { Component, Fragment } from "react";
```

Comment:
Looks like the `standardjs` did not run because we are using single quotes instead of double.

Can you please roll back these changes in this PR?

Reply:
Hi @joaopapereira . I rolled back the double quotes to be single quotes.